### DISCUSSION TITLE
OSMAnd on iOS not respecting conditional access

### DISCUSSION THREAD

**Discussion by NickKatchur** [osmandapp/OsmAnd#17654] [Maps Related]
I have added a conditional access tag to some interstates with major closure construction going on. I used the following tag as stated in other discussions and on the osm wiki, but it's not being recognized when doing any routing.

car:yes + car:conditional=no @ (2023 Jul 07 - 2023 Jul 29)

**Comment by dmpr0** [comment]
Please provide coordinates of the road. If it was added in July, it will be available only in August maps or you need you use Live updates.

**Reply by NickKatchur** [reply]
39°42′03.8″ N, 86°07′22.0″ W

I have live updates and am sure the map is getting updated. Someone recently updated it to be two separate tags using “motor_vehicle” but it is still not routing properly.

https://www.osm.org/way/618975524/history

**Reply by vshcherb** [reply]
We have issue with motor_vehicle directional support open, so this might be not related to osmand live
